M2Mqtt/Net/MqttNetworkChannel.cs

```diff
@@ -320,14 +320,10 @@ public static SslProtocols ToSslPlatformEnum(MqttSslProtocols mqttSslProtocol)
             {
                 case MqttSslProtocols.None:
                     return SslProtocols.None;
-                case MqttSslProtocols.TLSv1_0:
```

Comment:
why removing them? they are still valid.

Reply:
https://github.com/nanoframework/nf-interpreter/pull/2989

mainly: 
![image](https://github.com/user-attachments/assets/428005c7-a41e-485c-a473-1d724577575d)

Reply:
I think @josesimoes  forgot to do the announcement...